docs/testbed/README.testbed.Testbed_v2.hld.md

```diff
@@ -0,0 +1,127 @@
+# Testbed V2 Design
+# High Level Design Document
+
+
+# Table of Contents
+  * [About this Manual](#about-this-manual)
+  * [Requirement Overview](#Requirement-Overview)
+  * [Components](#Components)
+  * [Database Schema](#Database-Schema)
+  * [Implementation Plan](#Implementation-Plan)
+
+
+## About this Manual
+This documentation provides general information about the Testbed V2 feature implementation for `sonic-mgmt`. Testbed V2 aims to support multi-DUTs deployment with dynamic VLAN assignment and inter-DUTs link state propagation.
+
+## Requirement Overview
+* Maintain compatibility with existing testbed.
+* Support testbed/topology with multiple DUTs that have inter-DUTs connections.
+* Support flexible VLAN assignment.
+
+## Components
+* `test user`
+* `connection_db`
+  * Redis databases running on certain test server hosting connection-related metadata.
+* `servercfgd`
+  * running on the same server as `connection_db`
+  * a RPC server that is responsible for `connection_db` initial setup and provision.
+* `labcfgd`
+  * running on root/fanout switches
+  * subscribe to keyspace events of `connection_db` and act accordingly.
+
+## Database Schema
+* `LAB_CONNECTION_GRAPH_VERSIONS`
+```
+; ZSET that stores md5sum values of connection graph files that are used to provision `connection_db`
+key                              = LAB_CONNECTION_GRAPH_VERSIONS
```

Comment:
I'm still a little confused about what this is supposed to encode - is this the static view of how all the switches/servers/etc. are all physically connected or is this the current view of what the lab logically looks like (e.g. what topologies we have deployed, which DUTs are "neighbor" in the case of multi-DUT, etc.)?

Reply:
You mean `LAB_CONNECTION_GRAPH_VERSION`?

Reply:
Yes!

Reply:
I still don't get what this is describing. Is this 1) what version of the physical connections in the lab we are looking at or 2) what version of the logical connections in the lab we're looking at.

Reply:
This is the version of physical connections in the lab.